.github/workflows/analysis.yml

```diff
@@ -147,6 +147,14 @@ jobs:
               curve_plot.pdf
               fitted_curves.pdf
 
+      - name: Create Build Directory and Move PDFs
+        run: mv *.pdf website/
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.1
```

Comment:
How does this differ from [this](https://github.com/actions/upload-pages-artifact)? Using the official one seems to be the safer bet, if possible.